### ISSUE TITLE
Add option to generate spatial indexes for gpkg output

### ISSUE THREAD

**Issue by Chaz6** [hotosm/osm-export-tool-python#20]
It would be useful to be able to generate spatial indexes when generating a geopackage.

http://www.geopackage.org/spec/#extension_rtree

**Comment by bdon** [comment]
Agree, this would be useful. Ideally it should be an option for each table with a geometry column.
